packages/react-sdk/src/props.ts

```diff
@@ -147,54 +147,64 @@ export const resolveUrlProp = (
   if (instanceProps === undefined) {
```

Comment:
maybe worth adding a test, there are already some tests for this module

Reply:
Done, Thank you!